src/components/capture/Edit.tsx

```diff
@@ -115,7 +115,7 @@ function CaptureEdit() {
         exit: () => {
             resetState();
 
-            navigate(authenticatedRoutes.captures.path);
+            navigate(authenticatedRoutes.captures.fullPath, { replace: true });
```

Comment:
I feel like replacing edit is safer. That way you cannot click back and then try to save again and get an error.